src/sql/workbench/contrib/welcome/gettingStarted/browser/initialSetupWizard.css

```diff
@@ -0,0 +1,308 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.ads-initial-setup-wizard {
+	position: absolute;
+	margin: auto;
+	top: 0;
+	right: 0;
+	bottom: 0;
+	left: 0;
+	width: 719px;
+	z-index: 9;
+	background: transparent;
+}
+
+.ads-initial-setup-show {
+	display: block;
+}
+
+.ads-initial-setup-hide {
+	display: none;
+}
+
+.ads-initial-setup-wizard:focus {
+	outline: 0 none !important;
+}
+
+.ads-initial-setup-wizard-text-container {
+	padding: 0 24px 22px 24px;
+}
+
+.ads-initial-setup-wizard h1 {
+	font-size: 18px;
+	font-weight: 700;
+}
+
+.ads-initial-setup-wizard h2 {
+	font-size: 16px;
+	font-weight: 700;
+	line-height: 22px;
+	margin: 24px 0 12px;
+}
+
+.monaco-workbench > .ads-initial-setup-wizard {
+	position: absolute;
+	top: 0;
+	left: 0;
+	height: 100%;
+	width: 100%;
+	z-index: 9999;
+	font-size: 1em;
+	transition: font-size 0.25s;
+	outline: 0;
+	position: absolute;
+	z-index: 1;
+	background: rgba(77, 77, 77, 0.65);
+	opacity: 0;
+	transition: opacity 1s;
+	transition-duration: 1s;
+}
+
+.monaco-workbench > .ads-initial-setup-wizard--open {
```

Comment:
nit - but I feel like this class name is a bit verbose. Why not just have an "open" class and the use the selector `.ads-initial-setup-wizard.open`?